### DISCUSSION TITLE
Specify a custom tsconfig

### DISCUSSION THREAD

**Discussion by ramiel** [vercel/vercel#4832] [Help] [answered]
I'm building a project wiht node and I'd like to use a different `ŧsconfig.json` file for vercel builds only. Is this possible?

**Comment by styfle** [comment, marked as answer]
Vercel will detect the `tsconfig.json` file in the same directory as your `package.json` file

**Reply by ramiel** [reply]
Yes, ok. My question is if I can use a different tsconfig file, for example `tsconfig.vercel.json` instead of the normal one.

**Reply by styfle** [reply]
No, you would need to do the opposite: use `tsconfig.json` for Vercel and `tsconfig.other.json` for some other build system.

Something like `tsc --project tsconfig.other.json src/*.ts`

https://www.typescriptlang.org/docs/handbook/compiler-options.html

**Reply by ramiel** [reply]
Ok, thank you.

**Reply by NWYLZW** [reply]
Usually, the IDE will also detect this file and use it as a configuration for the project, but I don't want 'vercel dev' to also use this file, such as' tsconfig paths'.
Can't Vercel specify the path for 'tsconfig. json'?
